crates/miden-objects/src/transaction/mod.rs

```diff
@@ -3,19 +3,27 @@ use super::note::{NoteId, Nullifier};
 use super::vm::AdviceInputs;
 use super::{Felt, Hasher, WORD_SIZE, Word, ZERO};
 
+mod advice_inputs;
 mod executed_tx;
 mod inputs;
+mod kernel;
+mod kernel_procedures;
+pub mod memory;
 mod ordered_transactions;
 mod outputs;
 mod partial_blockchain;
 mod proven_tx;
 mod transaction_id;
 mod tx_args;
+mod tx_event_id;
 mod tx_header;
 mod tx_summary;
 
+pub use advice_inputs::TransactionAdviceInputs;
 pub use executed_tx::{ExecutedTransaction, TransactionMeasurements};
 pub use inputs::{AccountInputs, InputNote, InputNotes, ToInputNoteCommitments, TransactionInputs};
+pub use kernel::TransactionKernel;
+pub(crate) use kernel_procedures::KERNEL_PROCEDURES;
```

Comment:
This could be just `use kernel_procedures::KERNEL_PROCEDURES;` - right?

But also, I'm not sure we need this line at all (AFAICT, this is used only in a single place, and there, we could just import using a full path).